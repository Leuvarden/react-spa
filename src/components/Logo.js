import React from 'react';

export default React.createElement('div', {class: 'logo'}, 'netflixroulette')