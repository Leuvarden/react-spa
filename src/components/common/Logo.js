import React from 'react';
import PropTypes from 'prop-types';
import { NavLink } from 'react-router-dom';
import './../../styles/search-panel.scss';

const Logo = ({ path }) => {
    return (
        <div className="logo logo__container">
            <div className="logo__inner">
                netflixroulette
                {path.includes('film') ? 
                    <NavLink to="/" className="search-panel__button-search">Search</NavLink>
                    : null}
            </div>
        </div>
    );
};

export default Logo;

Logo.propTypes = {
    path: PropTypes.string
};

Logo.defaultProps = {
    path: ''
};