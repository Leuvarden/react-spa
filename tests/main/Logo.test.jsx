
import React from 'react';
import Logo from './../../src/components/main/Logo.js';


/* eslint-disable no-undef */
describe ('Logo', () => {

    it('renders as expected', () => {
        const logo = shallow(<Logo />);
    
        expect(logo).toMatchSnapshot();
    });

});