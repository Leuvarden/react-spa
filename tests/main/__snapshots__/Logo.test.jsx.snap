// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo renders as expected 1`] = `
<div
  className="logo"
>
  netflixroulette
</div>
`;
